spaces/domains.json

```
@@ -6,6 +6,7 @@
   "yearn.snapshot.page": "yearn",
   "signal.aave.com": "aave",
   "gov.xfinance.io": "xfinance",
+  "vote.nftgoddess.art": "nftgoddess",
```

Your domain field value should be similar to your space directory `"my.custom.domain": "my-space.eth"`.
```suggestion
  "vote.nftgoddess.art": "nftgoddess.eth",
```

---

I've just update the correct ens.